### PR TITLE
feat: Helm Repo

### PR DESCRIPTION
**Pull request by aexei**
### What change does this PR introduce?

Use the branch charts as a helm repo.

### Why was this change needed?

This is how to install novu via helmet without cloning the github repo

### Other information (Screenshots)

Please create an empty branch "charts".
The branch could additionally be published via github pages.

### CODE REVIEW

File: .github/workflows/release-helm.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: stefanprodan/helm-gh-pages@v1.7.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          charts_dir: "docker/kubernetes"
+          branch: charts
```
Comment (p-fernandez) [inline .github/workflows/release-helm.yml:18]: Does the branch has to be called `charts`? If the name could be chosen I'd go for `helm`.

File: docker/kubernetes/helm/templates/api/deployment.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: {{ printf "%s-%s" .Release.Name "api"  }}
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
-    app.kubernetes.io/component: novu-api
+    app.kubernetes.io/component: {{ printf "%s-%s" .Release.Name "api"  }}
```
Comment (p-fernandez) [inline docker/kubernetes/helm/templates/api/deployment.yaml:7]: These changes belongs to other PR.

**Review by p-fernandez — CHANGES_REQUESTED** [review]
I really liked you separated all the changes in different PRs (even if minimal changes) but helps to people like me less familiarised with Helm. Though this PR includes 2 different PRs.
Also I have a question regarding the branch suggested to create.
One thing regarding that is that we would need to add a workflow in the pipeline to keep `helm` aligned with either `main` or `next` branch, depending on what are the expectations to have with Helm.

File: .github/workflows/release-helm.yml (hunk 1 above)
Comment (aexei) [reply .github/workflows/release-helm.yml:18]: yes that doesn't matter, most use charts

File: docker/kubernetes/helm/templates/api/deployment.yaml (hunk 2 above)
Comment (aexei) [reply docker/kubernetes/helm/templates/api/deployment.yaml:7]: oh crap that was not intentional, apparently I made a mistake. Should I open a new PR or can you pick it up only in part?

Comment (p-fernandez) [reply docker/kubernetes/helm/templates/api/deployment.yaml]: Fixed. 😄

**Review by p-fernandez — APPROVED** [review]

**Pull request merged**